src/API/movies.js

```diff
@@ -40,4 +40,22 @@ export const UpdateMovie = async (obj) => {
         headers: { "Content-Type": "application/json" },
         body: JSON.stringify(obj),
     })
+}
+
+export const GetWatchlist = () => {
+    return fetch(`${url}/watchlist`)
+        .then((resp) => {
+            return resp.json();
+        })
```

Comment:
can we do the next step and this in a single promise?
Also add a catch block for error logging.

Reply:
Converting to .json take some time to resolve, so i dont think that we can do this in single promise. Catch block Done